### ISSUE TITLE
Crash After Dragging and Dropping Audio into Project

### ISSUE THREAD

**Issue by owlboy** [oel-mediateam/storybook-packager-public#17]
Crashed after dragging and dropping in some SVG files, then some MP3 files, and then attempting to click on a single entry in the page list.
[sbp-crashlog.txt](https://github.com/oel-mediateam/storybook-packager-public/files/3229383/sbp-crashlog.txt)

**Comment by owlboy** [comment]
Here is the presentation as it stands. I can't import files into it at all now.

[apc480_lesson8.sbproj.zip](https://github.com/oel-mediateam/storybook-packager-public/files/3229406/apc480_lesson8.sbproj.zip)

**Comment by Lin87** [comment]
@owlboy is the presentation project suppose to be empty? I opened it and it's empty (with the starter first page). I am able to drag and drop SVG and audio.

**Comment by owlboy** [comment]
No, it's not, it's crashing when I add stuff. I guess the crash log is all you have then. Once I made a new project it worked fine.

**Comment by Lin87** [comment]
I am not able to reproduce this crash. From the crash log, the crash was caused by a `Fatal error: Unexpectedly found nil while unwrapping an Optional value` by the `setImage()` method in `ImageAudioViewController` class. However, the `setImage()` method and its call to `setAudio()` method already have error handling for `nil`. In fact, `setAudio()` method has two error handling statements. One to check for nil, and the other one to check for successful reading. If the checks failed, they failed quietly and should not crash the app because there are no more lines of code to execute.

My guess might be something to do file system read/write permission.
